lib/rbac/filterer.rb

```diff
@@ -147,8 +147,10 @@ def self.accessible_tenant_ids_strategy(klass)
     #   - Array<Numeric> list if ids. :class is required. results are returned as ids
     #   - Array<Object> list of objects. results are returned as objects
     # @option options :named_scope   [Symbol|Array<String,Integer>] support for using named scope in search
-    #     Example without args: :named_scope => :in_my_region
-    #     Example with args:    :named_scope => [in_region, 1]
+    #     Example one scope without args:     :named_scope => :in_my_region
+    #     Example one scope with args:        :named_scope => [[:in_region, 1]]
```

Comment:
Unfortunately I'm not aware of a way, how to do it nicer for one scope with parameters. Since scope can be identified by both symbol and string (and when we would be pushing scopes via JSON, it'll be string) and parameters can be strings as well, we need the nested Array to distinguish between more chained scopes and one scope with parameters.

Reply:
I think this is great, we just need to separate this to another PR, so core guys can review this easily